crates/orderbook/src/database.rs

```diff
@@ -1,5 +1,6 @@
 pub mod app_data;
 pub mod auctions;
+mod fee_policy;
```

Comment:
Any reason this is the only module that's using singular (c.f auction**s**, trade**s**, etc).

Reply:
Renamed.